libopae/CMakeLists.txt

```diff
@@ -69,6 +72,13 @@ target_link_libraries(opae-c
   ${libuuid_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT})
 
+if (USE_NUMA)
+    find_package(NUMA REQUIRED)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -lnuma -I${NUMA_INCLUDE_DIR}")
```

Comment:
Can't we use `include_directories(${NUMA_INCLUDE_DIR})` instead?
Also, line 79 is redundant with `-lnuma` in this line

Reply:
Yes.  Done.